diff --git a/desktop/css/styles.pcss b/desktop/css/styles.pcss
index 2001d83..bd98d0c 100644
--- a/desktop/css/styles.pcss
+++ b/desktop/css/styles.pcss
@@ -19,61 +19,31 @@
 
 	.banner {
 		display: flex;
-		flex-direction: column;
-		font-size: 14px;
 
 		font-family: 'Arial', 'Helvetica', 'Verdana', sans-serif;
 		box-shadow: 0 3px 5px rgba( 0, 0, 0, 0.5 );
 		background-color: #ffffff;
 	}
 
-	.banner__content {
-		margin-top: 0.5em;
-	}
-
-	.banner__footer {
-		padding: 0.3em 0 0 0;
-		font-size: 1.1em;
-		color: #808080;
-	}
-
-	.banner > .banner__footer {
-		display: none;
-	}
-
 	.infobox {
 		display: flex;
 		flex-direction: column;
 		justify-content: space-between;
 		flex: 0 1 100%;
 
-		border: 0 none;
+		border: 5px solid #990000;
 		border-radius: 9px;
-		background-color: #0166a5;
+		background-color: #f8f9fa;
 		padding: 0.25em 0.5em 0.25em 0;
-		margin: 0.2em 5em 0.2em 5em;
-
-		position: relative;
-	}
-
-	.infobox:before {
-		position: absolute;
-		bottom: 1.5em;
-		content: ' ';
-		margin-left: -25px;
-		border-style: solid;
-		border-width: 0 0 25px 25px;
-		border-color: transparent transparent #0166a5 transparent;
+		margin: 0.2em 0 0.2em 0.2em;
 	}
 
 	.infobox__text {
-		color: #ffffff;
-		font-family: serif;
-		font-size: 1.2em;
+		color: #000000;
+		font-size: 1.05em;
 		line-height: 1.2em;
 		text-align: left;
-		margin: 0.5em 1em;
-
+		margin: 0 0 0.5em 2em;
 	}
 
 	.text__headline {
@@ -95,11 +65,10 @@
 	}
 
 	.text__paragraph .text__highlight {
-		border: 0 none;
+		border-bottom: solid 3px #990000;
 		padding: 0 0.1em;
-		background: #ffdd5b;
+		background: transparent;
 		font-weight: bold;
-		color: #000000;
 	}
 
 	/* This is for keeping the same banner size when testing different banner texts.
@@ -111,7 +80,7 @@
 
 	.infobox__footer {
 		padding: 0.025em 0 0 2em;
-		background: url( 'https://upload.wikimedia.org/wikipedia/donate/3/37/Information-icon-yellow.png' ) no-repeat left 0.5em top 0.1em/18px 18px;
+		background: url( 'https://upload.wikimedia.org/wikipedia/donate/9/99/RedInfoI.svg' ) no-repeat left 0.5em top 0.1em/18px 18px;
 	}
 
 	.footer {
@@ -122,7 +91,6 @@
 
 		font-size: 0.8em;
 		line-height: 1.6em;
-		color: #808080;
 	}
 
 	.footer__info {
@@ -147,72 +115,39 @@
 	}
 
 	.application-of-funds-link {
-		color: #808080;
+		color: #000000;
 	}
 
 	/* Form styling */
 
 	.form {
-		order: 2;
-		width: auto;
-		min-width: inherit;
+		width: 16em;
+		min-width: 12em;
 		flex-shrink: 0;
-		padding: 0.2em 0 0 0.5em;
-		margin: 0 5em;
-	}
-
-	#WMDE_Banner-form {
-		display: flex;
-		flex-direction: row;
-		flex-wrap: wrap;
-	}
-
-	.form__header {
-		display: flex;
-		justify-content: space-between;
-		color: #97b314;
-		width: 100%;
-		font-weight: bold;
-	}
-
-	.form-field-group {
-		display: flex;
-		flex-wrap: nowrap;
-		flex: 1 1 auto;
-	}
-
-	.select-group-container {
-		display: flex;
-		flex: 1 1 auto;
-		flex-direction: column;
-		margin-right: 0.5em;
+		padding: 1em 0 0 0.5em;
+		margin: 0.5em 0 0.5em 0.25em; /* Same as infobox, but not to the right */
 	}
 
 	.select-group {
 		border: 0;
 		margin: 0 auto;
-		padding: 0.4em 0.5em 0 0.5em;
+		padding: 0 0.5em 0.5em 0.5em;
 		display: flex;
-		flex-wrap: nowrap;
+		flex-wrap: wrap;
 		flex-direction: row;
 		justify-content: flex-start;
-		flex: 1 0 auto;
+		flex: 1 1 100%;
 		font-size: 0.9em;
 		font-weight: bold;
-		width: 100%;
 	}
 
-	.select-group:first-child {
-		padding-left: 0;
-	}
-
-	.select-group:last-child {
-		padding-right: 0;
+	.select-group--with-error {
+		background: #990000;
 	}
 
 	.select-group__errormessage {
 		display: none; /* Message will be shown by JS code */
-		color: #000000;
+		color: #ffffff;
 		font-weight: bold;
 		text-align: center;
 		width: 100%;
@@ -220,37 +155,28 @@
 		font-size: 0.74em;
 	}
 
-	.select-group-container--with-error .select-group__errormessage {
-		background: #990000;
-		color: #ffffff;
-	}
-
 	.select-group__input {
 		display: none;
 	}
 
 	.select-group__custom-input {
-		background: #e5ecc4;
-		color: #97b314;
 		padding: 0.4em;
 		width: 100%;
-		height: inherit;
+		height: 100%;
 		margin: 0;
-		font-size: 0.9em;
-		font-weight: bold;
+		font-size: 0.74em;
 		text-align: center;
-		border: 0 none;
+		border: 1px solid #9aa0a7;
 		border-radius: 4px;
 		box-sizing: border-box;
 	}
 
 	.select-group__option {
 		flex: 1 1 auto;
-		padding: 0 0.2em 0.2em 0;
+		padding: 0 0.3em 0.3em 0;
 		margin: 0;
 		box-sizing: border-box;
 		display: block;
-		position: relative;
 	}
 
 	.select-group__option--fullwidth {
@@ -261,10 +187,6 @@
 		width: 50%;
 	}
 
-	.select-group__option--thirdwidth {
-		width: 33%;
-	}
-
 	.select-group__option--quarterwidth {
 		width: 25%;
 	}
@@ -275,61 +197,34 @@
 
 	.select-group__state {
 		display: block;
-		background: #e5ecc4;
-		color: #97b314;
-		border: 0 none;
-		padding: 0 0.4em;
+		background: #f8f9fa;
+		border: 1px solid #9aa0a7;
+		border-radius: 2px;
+		padding: 0.1em 0;
 		line-height: 2em;
 		cursor: pointer;
 		text-align: center;
-		font-size: 0.95em;
+		font-size: 0.8em;
 		box-sizing: border-box;
-		white-space: nowrap;
-	}
-
-	.select-group__custom-input + .select-group__state {
-		position: absolute;
-		top: 0;
-		right: 1em;
-		border-color: transparent;
-		background: transparent;
 	}
 
 	.select-group__input:checked + .select-group__state {
-		color: #e5ecc4;
-		background-color: #97b314;
-		border: 0 none;
-		outline: 0 none;
-		border-radius: 0;
-
-	}
-
-	.select-group__option:first-child .select-group__state {
-		border-radius: 6px 0 0 6px;
-	}
-
-	.select-group__option:last-child .select-group__state,
-	.select-group__custom-input {
-		border-radius: 0 6px 6px 0;
-	}
-
-	.select-group__custom-input--value-entered {
-		color: #e5ecc4;
-		background-color: #97b314;
-		border: 0 none;
-		outline: 0 none;
+		color: #000000;
+		background: #f8f9fa;
+		border: 1px solid #2a4b8d;
+		outline: 2px solid #2a4b8d;
 		border-radius: 0;
 	}
 
 	.button-group {
-		display: block;
 		border: 0;
-		margin: 0;
-		padding: 0.3em 0.5em 0.3em 0;
+		margin: 0 auto;
+		padding: 0 0.5em;
+		display: flex;
 		flex-wrap: wrap;
 		flex-direction: row;
 		justify-content: flex-start;
-		flex: 0 1 auto;
+		flex: 1 1 100%;
 	}
 
 	.button-group__button {
@@ -343,35 +238,44 @@
 		border: 0 none;
 		background: transparent;
 		font-size: inherit;
-		width: 100%;
 	}
 
 	.button-group__button--halfwidth {
 		width: 50%;
 	}
 
+	@media ( max-width: 1100px ) {
+		.button-group__button--halfwidth .button-group__label {
+			padding: 0.3em 0.1em;
+		}
+	}
+
+	@media ( max-width: 1100px ) {
+		.button-group__button--halfwidth {
+			width: 100%;
+		}
+	}
+
 	.button-group__label {
 		display: block;
-		color: #e5ecc4;
-		background: #97b314;
-		border: 0 none;
-		border-radius: 6px;
-		padding: 0 0.4em;
+		color: #ffffff;
+		background: #2a4b8d;
+		border: 1px solid #9aa0a7;
+		border-radius: 2px;
+		padding: 0.1em 0;
 		line-height: 2em;
-		font-size: 0.85em;
+		font-size: 0.7em;
 		font-weight: bold;
 		text-align: center;
 		box-sizing: border-box;
-		white-space: nowrap;
 	}
 
 	.close {
-		position: absolute;
 		width: 18px;
 		margin: 0.25em 0.25em 0.25em 0;
-		top: 0.5em;
-		right: 0.5em;
-		opacity: 0.5;
+		top: auto;
+		right: auto;
+		flex: 0 0 18px;
 	}
 
 	.click-tracking {
@@ -385,124 +289,7 @@
 	}
 
 	.progress_bar__days_left {
-		display: block;
-		color: #ffffff;
-		line-height: 1.9em;
-	}
-
-	.progress_bar__wrapper {
-		border-color: #dce4e8;
-		background-color: #ffffff;
-		margin: 0.5em 1em;
-	}
-
-	.progress_bar__donation_fill {
-		background: #97b314;
-		height: 1.6em;
-	}
-
-	.progress_bar__donation_text {
-		color: #ffffff;
-		line-height: 1.8em;
-	}
-
-	.progress_bar__donation_remaining {
-		color: #97b314;
-		line-height: 1.8em;
+		display: none;
 	}
 
-	@media ( min-width: 1280px ) {
-		.banner__content {
-			display: flex;
-			flex-direction: row;
-			order: 1;
-		}
-
-		.banner__footer {
-			order: 2;
-			padding: 0.5em 0 0 6.5em;
-		}
-
-		.banner > .banner__footer {
-			display: block;
-		}
-
-		.infobox {
-			margin: 0.2em 0 0.2em 5em;
-		}
-
-		.form {
-			margin: 0 1.5em 0 0.25em;
-			padding-top: 0;
-			width: 18em;
-		}
-
-		.form__header > p {
-			width: 100%;
-		}
-
-		.form__header > .banner__footer {
-			display: none;
-		}
-
-		.form > div {
-			width: 100%;
-		}
-
-		.form__header {
-			text-align: center;
-		}
-
-		.form-field-group {
-			display: block;
-		}
-
-		.select-group,
-		.button-group {
-			flex-wrap: wrap;
-		}
-
-		.select-group:first-child {
-			padding-left: 0.5em;
-		}
-
-		.select-group:last-child {
-			padding-right: 0.5em;
-		}
-
-		.select-group-container--with-error .select-group__errormessage {
-			margin-left: 0.5em;
-		}
-
-		.WMDE-Banner-amounts {
-			.select-group__option:nth-child( 1 ) .select-group__state {
-				border-radius: 6px 0 0 0;
-			}
-
-			.select-group__option:nth-child( 4 ) .select-group__state {
-				border-radius: 0 6px 0 0;
-			}
-
-			.select-group__option:nth-child( 5 ) .select-group__state {
-				border-radius: 0 0 0 6px;
-			}
-
-			.select-group__custom-input {
-				border-radius: 0 0 6px 0;
-			}
-		}
-
-		.button-group {
-			margin-left: 0.5em;
-			margin-right: -0.5em;
-		}
-
-		#WMDE_Banner-form {
-			display: block;
-		}
-
-		.footer__item--rightflex {
-			margin-right: 5em;
-		}
-	}
 }
